core/src/main/java/com/sequenceiq/cloudbreak/service/LoadBalancerConfigService.java

```diff
@@ -119,6 +135,8 @@ public String generateLoadBalancerEndpoint(Stack stack) {
         switch (targetGroup.getType()) {
             case KNOX:
                 return Set.of(new TargetGroupPortPair(Integer.parseInt(httpsPort), Integer.parseInt(knoxServicePort)));
+            case OOZIE:
+                return Set.of(new TargetGroupPortPair(Integer.parseInt(OOZIE_HTTPS_PORT), Integer.parseInt(OOZIE_HTTPS_PORT)));
```

Comment:
Probably a dumb question. In the situation where the LB is shared (public / private) - does this end up opening up external access to Oozie endpoints via the LB (as against via Knox) (Was this port otherwise blocked by network security rules?)

Reply:
These ports are always blocked by network security rules. Only the cluster nodes can communicate with each other through the Oozie https port.